Composer/packages/lib/shared/src/lgUtils/models/stringTypes.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+/**
+ * example: '- [bfdactivity-123456]'
```

Comment:
For now, i think we should leave the trick of "-" of this abstraction.

Reply:
Seems I missed one thing here. In `LgEditorWidget.ts` line 16, there appears the '- hello' format. The type should be:
```
type LgBody = string; // '- hello'
type LgTemplateRefString = string; // '[greeting]'
type LgText = LgBody | LgTemplateRefString;
```
@boydc2014

Reply:
what i mean is if we think lgText is what been filled into json like 'acitivity', 'prompt', etc. in there, there is no "-". 

The trick been done in Form, is when lgBody don't begin with a "-", attach a "-" as prefix, i mean such logic don't have to be included here. 

To be more specific, when you create a SendActivity,
the json which looks like 
```
dialog.json
{
   $type: "SendAcitivity"
   activity: "[bfactivity-1234]" // if we call this lgText?
}

a.log
# bfactivity-1234
- hello world // this is lg body, we can think that lg body is  mutiple ("-" + lgText) 
```

people may change the body to 

```
#bfactivity-1234
- @{another-template()} 
- @{another-template2()} hello world
```

So, the point is lgBody is list of ("-" + lgText), and we don't have to include lgBody concept in this centralization.